### PR TITLE
Enhancement/gcp iam 1.9 1.10

### PR DESCRIPTION
**Pull request by SophieDorval**
Added rules 1.9 and 1.10 from GCP cis

**Make sure the PR is against the `develop` branch (see [Contributing](https://github.com/nccgroup/ScoutSuite/blob/master/CONTRIBUTING.md)).**

Please include a summary of the change(s) and which issue(s) it addresses. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Select the relevant option(s):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ScoutSuite/output/data/html/partials/gcp/services.kms.projects.id.keyrings.html [hunk 1]
```diff
@@ -17,20 +17,43 @@ <h4 class="list-group-item-heading"><span id="kms.projects.{{@../key}}.keyrings.
...
+                        <li id="kms.projects.{{@../../key}}.keyrings.{{@../key}}.keys.{{@key}}.state">State: <samp>{{state}}</samp></li>
                         <li id="kms.projects.{{@../../key}}.keyrings.{{@../key}}.keys.{{@key}}.protection_level">Protection Level: {{protection_level}}</li>
                         <li id="kms.projects.{{@../../key}}.keyrings.{{@../key}}.keys.{{@key}}.algorithm">Algorithm: <samp>{{algorithm}}</samp></li>
                         <li id="kms.projects.{{@../../key}}.keyrings.{{@../key}}.keys.{{@key}}.purpose">Purpose: {{purpose}}</li>
                         <li id="kms.projects.{{@../../key}}.keyrings.{{@../key}}.keys.{{@key}}.creation_datetime">Creation Date: {{format_date creation_datetime}}</li>
-                        <li id="kms.projects.{{@../../key}}.keyrings.{{@../key}}.keys.{{@key}}.rotation_period">Rotation Period: {{value_or_none rotation_period}}</li>
+                        <li id="kms.projects.{{@../../key}}.keyrings.{{@../key}}.keys.{{@key}}.rotation_period">Rotation Period: {{value_or_none rotation_period}}s</li>
```
Comment (j4v) [inline ScoutSuite/output/data/html/partials/gcp/services.kms.projects.id.keyrings.html:25]: `None`s?

File: ScoutSuite/providers/gcp/rules/rulesets/default.json [hunk 2]
```diff
@@ -242,6 +242,18 @@
                 "level": "warning"
             }
         ],
+        "kms-cryptokeys-anonymously-publicly-accessible.json": [
+            {
+                "enabled": true,
+                "level": "warning"
```
Comment (j4v) [inline ScoutSuite/providers/gcp/rules/rulesets/default.json:248]: `danger`

File: ScoutSuite/providers/gcp/rules/rulesets/cis-1.1.0.json [hunk 3]
```diff
@@ -7,6 +7,18 @@
               "level": "warning"
           }
       ],
+      "kms-cryptokeys-anonymously-publicly-accessible.json": [
+        {
+              "enabled": true,
+              "level": "warning"
```
Comment (j4v) [inline ScoutSuite/providers/gcp/rules/rulesets/cis-1.1.0.json:13]: `danger`

File: ScoutSuite/output/data/html/partials/gcp/services.kms.projects.id.keyrings.html (hunk 1 above)
Comment (SophieDorval) [reply ScoutSuite/output/data/html/partials/gcp/services.kms.projects.id.keyrings.html]: Probably a mistake. It is removed

File: ScoutSuite/providers/gcp/rules/rulesets/default.json (hunk 2 above)
Comment (SophieDorval) [reply ScoutSuite/providers/gcp/rules/rulesets/default.json]: Changed warning for danger

File: ScoutSuite/providers/gcp/rules/rulesets/cis-1.1.0.json (hunk 3 above)
Comment (SophieDorval) [reply ScoutSuite/providers/gcp/rules/rulesets/cis-1.1.0.json]: Changed warning for danger

**Review by j4v — APPROVED** [review]

**Pull request merged**